jdk/test/sun/nio/cs/TestIBMBugs.java

```diff
@@ -22,8 +22,9 @@
  */
 
 /* @test
-   @bug 6371437 6371422 6371416 6371619 5058184 6371431 6639450 6569191 6577466
-   @summary Check if the problems reported in above bugs have been fixed
+ * @bug 6371437 6371422 6371416 6371619 5058184 6371431 6639450 6569191 6577466 8235834
+ * @summary Check if the problems reported in above bugs have been fixed
+ * @modules jdk.charsets
```

Comment:
Please make sure that you're only backporting the "changes" done by the head-stream patch. Not more than that (if not required). See my comments on #480 .